.release/versions.hcl

```diff
@@ -6,6 +6,9 @@
 
 schema = 1
 active_versions {
+  version "1.20" {
+    ce_active = true
+  },
   version "1.19" {
     ce_active = true
```

Comment:
I think we'd need to remove the `ce_active` from this line correct?

Reply:
The engineering release documents explicitly state not to remove the ce_active line from the previous version at this stage, as that will happen in a post-release stage

Reply:
<img width="969" alt="Screenshot 2024-09-16 at 3 37 09 PM" src="https://github.com/user-attachments/assets/0f9d6070-1027-43ad-abbc-abebf03ed50e">

Reply:
FYI, think we need to remove the comma at the end of that block (BPA is unhappy about it): https://github.com/hashicorp/consul/pull/21752

Mostly commenting in case of future copypasta PRs - at least that's how I often remind myself of release steps 😄